### PR TITLE
Interview with Erik Veerman

### PR DESCRIPTION
**Pull request by AlbaRoza**
Interview in Dutch and English with Erik Veerman from 's-Hertogenbosch.

### CODE REVIEW

File: src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Het grootste voordeel van replicatie van Signalen is dat **veel van de gewenste functionaliteit al is gerealiseerd en bewezen in Amsterdam**. Als je vanaf nul begint, is er veel meer onzekerheid of en hoe de oplossing zal eindigen. En replicatie heeft waarschijnlijk een financieel voordeel, hoewel ik niet zeker weet of dit waar is.
+
+4. Wat waren de belangrijkste uitdagingen?
+
+De twee grootste uitdagingen waren:
+     a. Vind een balans tussen ons uitgebreide eisen- en wensenlijstje en de nog te ontwikkelen of aan te passen functionaliteit.
+     b. Zoveel mogelijk onderdelen en functionaliteit van SIA geschikt maken voor gebruik in 's-Hertogenbosch (en andere gemeenten) en toch blijven fuseren tot één Signalen masterbranche zodat alle Signalen deelnemers elkaar kunnen ontwikkelen. Vooral in het begin was er spanning met Amsterdam omdat Signalen (SIA) hun creatie is en aanpassingen aan Signalen door of voor anderen dan als moeilijk ervaren kunnen worden.
+    
+5. Toekomstperspectieven voor 's-Hertogenbosch
```
Comment (felixfaassen) [inline src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md:35]: Toekomstperspectieven voor Signalen in 's-Hertogenbosch

File: src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Wat Signalen betreft hebben we nog een uitgebreide lijst met gewenste functionaliteiten. Deze willen we het liefst samen met andere gemeenten ontwikkelen.
+
+6. Voordelen van het gebruik van open source-technologieën in een replicatie als deze
+
+Goede vraag! Ik weet het niet zeker. Maar ik denk dat het makkelijker is om met open source technologie een duurzame omgeving/gemeenschap van softwareleveranciers en andere gemeenten te creëren.
+
+7. Wat zou je veranderen als je helemaal opnieuw zou beginnen?
+
+In het begin hadden we niet echt een idee hoe Signalen werkte, wat de mogelijkheden waren en wat de achterliggende gedachten van bepaalde keuzes waren. Hierdoor was het moeilijk om te bepalen wat Signalen van onze eisen en wensen te bieden had. Dit veroorzaakte ook enkele problemen bij het testen van de applicatie. Bij de aanschaf van een softwareoplossing krijgt u ondersteuning van een soort implementatieadviseur. Als deze rol aan het begin van het traject bij Signalen beschikbaar was geweest, was het traject een stuk eenvoudiger geweest. Dus als ik het opnieuw zou kunnen doen, zou ik zo snel mogelijk iemand aan boord halen met (proces)kennis van Signalen.
```
Comment (felixfaassen) [inline src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md:45]: "wat Signalen **ten aanzien** van onze eisen en wensen te bieden had."

File: src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+8. Tips die je een volgende replicator zou geven
+
+Zie hierboven en probeer informatie en documentatie te krijgen die door andere gemeenten wordt gebruikt.
```
Comment (felixfaassen) [inline src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md:49]: probeer informatie en documentatie te krijgen die door andere gemeenten wordt gebruikt. 
Suggest "probeer informatie en documentatie in te winnen bij andere gemeenten die Signalen gebruiken"

File: src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Een van onze interne worstelingen was om onze IT-afdeling aan boord te krijgen. Er was weerstand omdat Signalen en het beheer van Signalen (beheer van Signalen) heel anders is dan de bestaande applicaties. Hoewel we veel tijd hebben gestoken in het betrekken van hen, was het proces nog steeds erg moeilijk en traag. Ik kan aanraden om de IT-afdeling in een vroeg stadium te betrekken en gedurende het hele proces betrokken te houden.
```
Comment (felixfaassen) [inline src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md:50]: remove "(beheer van Signalen)"  duplicate

**Review by felixfaassen — DISMISSED** [review]
Hey @AlbaRoza great article and kudos for you to write it in Dutch yourself. I've made some suggestions. Also have @ElenaFdR have a look at it if you want.

**Comment by Ainali** [comment]
The filenames need to change. For the language button to work the filenames must be the same except for the .en. That means we cannot translate the filename.

**Review by AlbaRoza — COMMENTED** [review]
Looking perfect!

File: src/pages/news/2021-06-11-signalen-by-first-replicator-interview-erik-veerman.en.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+3. Why is a replication better than starting from zero your own environment?
+
+The biggest of the advantages of replication of Signalen is that **a lot of the desired functionality has already been realized and proven in Amsterdam**. When you start from zero there is a lot more uncertainty if and how the solution is going to end up. And replication probably has an financial advantage, although I'm not sure if this is true. 
+
+4. What were the main challenges?
+
+The two biggest challenges were to: 
+    a. Find a balance between our extensive requirements and wishes list and the functionality yet to be developed or adapted.
+    b. Make as many parts and functionality of SIA suitable for use in 's-Hertogenbosch (and other municipalities) and still continue to merge to one Signalen master branche so all Signalen participants can each other developments. Especially in the beginning there was a tension with Amsterdam because Signalen (SIA) is their creation and adjustments to Signalen by or for others can then be experienced as difficult.
```
[the remark this answers is not in the document]
Comment (AlbaRoza) [reply src/pages/news/2021-06-11-signalen-by-first-replicator-interview-erik-veerman.en.md:31]: Thanks for noticing this, @felixfaassen however,  I think I'll leave his full answer for journalistic deontology :)

I think a nice idea would be to also interview Amsterdam, so we'd have their point of view.

**Review by AlbaRoza — COMMENTED** [review]
One last review?

File: src/pages/news/2021-06-21-interview-with-erik-veerman-first-replicator-signalen.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+---
+path: "/news/2021-06-21-interview-with-erik-veerman-first-replicator-signalen/".
```
Comment (Ainali) [inline src/pages/news/2021-06-21-interview-with-erik-veerman-first-replicator-signalen.md:2]:
```suggestion
path: "/news/2021-06-21-interview-with-erik-veerman-first-replicator-signalen/"
```

**Review by Ainali — APPROVED** [review]

**Comment by AlbaRoza** [comment]
After several reviews it's ready to go!

**Pull request merged**